src/node.cc

```diff
@@ -195,10 +195,27 @@ void SignalExit(int signo, siginfo_t* info, void* ucontext) {
 }
 #endif  // __POSIX__
 
+#if HAVE_INSPECTOR
+class InspectorPauseResetter {
+ public:
+  explicit InspectorPauseResetter(inspector::Agent* agent) : agent_(agent) {}
+  ~InspectorPauseResetter() {
+     if (agent_->options().break_node_first_line) {
+       agent_->PauseOnNextJavascriptStatement("Break at bootstrap");
+     }
+  }
+ private:
+  inspector::Agent* agent_;
+};
+#endif
+
 MaybeLocal<Value> ExecuteBootstrapper(Environment* env,
                                       const char* id,
                                       std::vector<Local<String>>* parameters,
                                       std::vector<Local<Value>>* arguments) {
+#if HAVE_INSPECTOR
+  InspectorPauseResetter pause_resetter{env->inspector_agent()};
```

Comment:
I don’t think you need a separate class for this:

```suggestion
  OnScopeLeave pause_resetter([&]() {
    inspetor::Agent* agent = env->inspector_agent();
    if (agent->options().break_node_first_line)
      agent->PauseOnNextJavascriptStatement("Break at bootstrap");
  });
```

Reply:
Ah, now that I see `OnScopeLeave` remember it :( That is much nicer! Thanks